### ISSUE TITLE
Support for Angular 12+

### ISSUE THREAD

**Issue by mhuffsti** [kzrfaisal/angular-file-uploader#103]
I've been using the angular-file-uploader in one project in order to compare against custom code to handle file uploads.
I recently (today) updated that project to Angular 12.2, and when running NPM (8.1.0), I am getting this message:
npm ERR! Found: @angular/common@12.2.13
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^12.2.12" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^10.0.5" from angular-file-uploader@7.1.1
npm ERR! node_modules/angular-file-uploader
npm ERR!   angular-file-uploader@"7.1.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

Would it be possible for you to update the root angular-file-uploader project to reference a higher version of Angular?

**Comment by tariqahsan** [comment]
Instead of using bootstrap can we use angular material?

**Comment by cenacrharsh** [comment]
True, I am using angular ver 13 currently and I am unable to use this package as it won't install.
